src/code/stores/simulation-store.coffee

```diff
@@ -101,6 +101,12 @@ SimulationStore   = Reflux.createStore
   onRunSimulation: ->
     @_runSimulation()
 
+  stepUnits: () ->
+    if @isRecordingPeriod
```

Comment:
Should this be `@settings.isRecordingPeriod`? Non-step time units don't seem to get sent, otherwise.